src/views/Accessibility/AccessibilityRequestDetailPage.test.tsx

```diff
@@ -469,49 +473,42 @@ describe('AccessibilityRequestDetailPage', () => {
           <AccessibilityRequestDetailPage />
         );
         render(providers);
+
+        await waitForElementToBeRemoved(() =>
+          screen.getByTestId('page-loading')
+        );
+
+        const textbox = screen.getByRole('textbox', { name: /note/i });
+        userEvent.type(textbox, 'This is very well done');
+        screen.getByRole('button', { name: 'Add note' }).click();
         expect(await screen.findByRole('alert')).toBeInTheDocument();
```

Comment:
To make this assertion better, you can store this element into a variable and use `within` and `getByText`. This way, the test isn't searching the entire document for text.

This is a common improvement that can be added to many of the tests in this PR.